views/components/table.html

```diff
@@ -0,0 +1,25 @@
+{% macro table(tableData) %}
+<article class="base--article" itemscope itemtype="http://schema.org/Table">
+  <header class="base--header">
+    <h1 class="base--h1" itemprop="name">{{tableData.name}}</h1>
+  </header>
+  <table class="base--table">
+    <thead class="base--thead">
+      <tr class="base--tr">
+        {% for header in tableData.data.headers %}
+        <th class="base--th">{{header}}</th>
+        {% endfor %}
+      </tr>
+    </thead>
+    <tbody class="base--tbody">
+        {% for row in tableData.data.row %}
+        <tr class="base--tr" itemprop="dataset" itemscope itemtype="https://schema.org/Dataset">
```

Comment:
ok, I think I understand what this shouldn't be, but I can't figure out what it should be. I moved the `itemscope` and `itemtype` to the `table` level and removed `itemprop` for now since I'm not sure what that should be.